templates/frag/fromMultipart.hbs

```diff
@@ -0,0 +1,42 @@
+{{!--
+Output a reference to a FromJsonFunction function.
```

Comment:
This comment needs to change...

```
Output a reference to a function to transform the multipart input value into a JavaScript value
```

We could change the comment in `fromJson.hbs` to be clearer too:

```
Output a reference to a function to transform the JSON input value into a JavaScript value
```